src/TestFramework/TestFramework/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,6 @@
 ﻿#nullable enable
+Microsoft.VisualStudio.TestTools.UnitTesting.ClassCleanupBehavior.EndOfAssembly = 1 -> Microsoft.VisualStudio.TestTools.UnitTesting.ClassCleanupBehavior
```

Comment:
Sorry my previous comment wasn't super clear. These changes should also stay in the shipped. Basically, any change of public API already shipped should stay in shipped.